src/stories/Library/pause-reservation/PauseReservation.stories.tsx

```diff
@@ -10,10 +10,14 @@ export default {
     isChecked: {
       defaultValue: true,
     },
-    text: {
+    pauseText: {
```

Comment:
Great making this naming more specific for the context. We love understanding reading code ❤️